.vscode/settings.json

```diff
@@ -1,3 +1,4 @@
 {
-  "deepscan.enable": true
+  "deepscan.enable": true,
+  "typescript.tsdk": "./code/node_modules/typescript/lib"
```

Comment:
This makes sure the language server in VSCode uses the same version as used by tsc. Without. this you will get syntax errors for satisfies in VSCode.